torch/csrc/jit/passes/quantization.cpp

```diff
@@ -637,8 +637,8 @@ class InsertQuantDeQuantHelper {
   std::unordered_map<Graph*, std::vector<std::string>>
       observer_modules_to_remove_;
   std::unordered_map<Graph*, std::vector<Node*>> nodes_to_destroy_;
-  std::unordered_map<Graph*, std::unordered_map<Value*, QParamMap>>
-      values_to_qparams_;
+  // Map from Graph to observer node
```

Comment:
That comment is not very helpful, it's basically repeats what's written on the line below. Please expand it a bit.